sources/packages/web/src/services/http/dto/EducationProgramOffering.dto.ts

```diff
@@ -114,6 +118,24 @@ export enum OfferingYesNoOptions {
   No = "no",
 }
 
+/**
+ * Aviation Yes/No options.
+ */
+export enum AviationYesNoOptions {
```

Comment:
Similar offering properties are using the `OfferingYesNoOptions`, is there a particular reason to define a type for `AviationYesNoOptions`?